specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -120,8 +120,11 @@
             },
             "description": "OK"
           },
-          "202": {
```

Comment:
Could you please help us understand why you remove this 202 response?

Reply:
@aim-for-better Raised PR #11690 for same.

Reply:
@xolve , thanks. but I saw that PR is still a draft? When will that PR be ready for a review?

Reply:
Updated the PR.